src/nvim/testdir/test_statusline.vim

```diff
@@ -9,6 +9,10 @@ source view_util.vim
 source check.vim
 source term_util.vim
 
+func SetUp()
+  set laststatus=2
+endfunc
```

Comment:
`Test_statusline()` also fails when run alone. I'll upstream this change to Vim.